### DISCUSSION TITLE
Is there some way to open the file in remote VCS?

### DISCUSSION THREAD

**Discussion by goetzc** [NeogitOrg/neogit#483] [Q&A]
Something like [tpope/vim-fugitive](https://github.com/tpope/vim-fugitive) plugin which has "plugins" such as [tpope/vim-rhubarb](https://github.com/tpope/vim-rhubarb) which allows to open the file selection in GitHub, [shumphrey/fugitive-gitlab.vim](https://github.com/shumphrey/fugitive-gitlab.vim) for GitLab, or [tommcdo/vim-fubitive](https://github.com/tommcdo/vim-fubitive) for Bitbucket.

Then it's possible to run `:GBrowse` to open the link the default browser.

Is something like that possible with `Neogit` build-in or as an additional extension?

**Comment by CKolkey** [comment]
Not currently possible, no.

**Comment by rbhanot4739** [comment]
What is the state of the feature asked in above question as of now ?

**Comment by peterbenjamin-kh** [comment]
@goetzc / @rbhanot4739  - This is now available via `o` keybinding on the status buffer ([1](https://github.com/NeogitOrg/neogit/issues/1327),[2](https://github.com/NeogitOrg/neogit/blob/9bb1e73c534f767607e0a888f3de4c942825c501/README.md?plain=1#L368)).

@CKolkey - would you be open to accepting a PR for yanking the URL (equivalent to fugitive's `:GBrowse!` command)?

**Reply by CKolkey** [reply]
Hell yeah :)
